index.php

```diff
@@ -0,0 +1,12 @@
+<?php
+require_once './functions.php';
+$db = new PDO('mysql:host=db; dbname=kantopokemon', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM `kanto`');
```

Comment:
would be better to call the table "kanto pokemon" (plural) so it's describing that each item is a kanto pokemon